README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+# React Redux Capstone Project (Bank Holidays in United Kingdom)
 
```

Comment:
- [ ] Please as it is a requirement for this project, kindly add a presentation video as specified in the screenshot below of the project requirement

![video](https://user-images.githubusercontent.com/65852621/143667969-e7efb075-c84b-4e5b-91a4-2a1071d69670.png)

- [ ] Please add a link to the video presentation in your `Readme` file.

Reply:
### I completely forgot about making a video. Let me create it. 😃